README.md

```diff
@@ -267,7 +266,7 @@ func initConfig() {
 		viper.SetConfigFile(cfgFile)
 	} else {
 		// Find home directory.
-		home, err := homedir.Dir()
+		home, err := os.UserHomeDir()
```

Comment:
This would force users to upgrade Go in order to use this package making this a potentially breaking change.

Reply:
This is supported in `>=1.12`. Versions `<=1.11` are deprecated. I think it is good to enforce users not to use non-maintained software.

Reply:
Forcing a breaking change is not something we are going to include in a 1.0.0 release.

Reply:
It's ok. At this point, I'm just waiting for any v1.0.0 to be released (not matter what is included), so that we can move get past the blocking factor that is keeping tens of PRs on hold for months/years.